src/middleware/auth.js

```diff
@@ -0,0 +1,82 @@
+const jwt = require('jsonwebtoken');
```

Comment:
random nit: I would say all the functions in this file except `authenticateToken` would better belong in a file like `src/services/auth.js`. but it's not a big deal